pom.xml

```diff
@@ -299,6 +299,32 @@ under the License.
           <artifactId>apache-rat-plugin</artifactId>
           <version>0.13</version>
         </plugin>
+        <!-- Disable execution of some plugins in m2e (https://www.eclipse.org/m2e/documentation/m2e-execution-not-covered.html) -->
+        <plugin>
+          <groupId>org.eclipse.m2e</groupId>
+          <artifactId>lifecycle-mapping</artifactId>
+          <version>1.0.0</version>
+          <configuration>
+            <lifecycleMappingMetadata>
+              <pluginExecutions>
+                <pluginExecution>
+                  <!-- no native m2e support yet (https://issues.apache.org/jira/browse/MRRESOURCES-85) -->
+                  <pluginExecutionFilter>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-remote-resources-plugin</artifactId>
+                    <versionRange>[1.7.0,1.8.0)</versionRange>
+                    <goals>
+                      <goal>process</goal>
+                    </goals>
+                  </pluginExecutionFilter>
+                  <action>
+                    <ignore/>
+                  </action>
+                </pluginExecution>
+              </pluginExecutions>
+            </lifecycleMappingMetadata>
+          </configuration>
+        </plugin>
```

Comment:
> putting this in a profile activated by the presence of the m2e.version property.

Thanks for the hint, done in https://github.com/apache/maven-apache-parent/pull/64/commits/607c15461c90b41552e05ad7e310002209d7c6eb.

> This is useful to suppress no matter what version of the plugin is used. So, you can just specify a versionRange of [0,)

I disagree, I enabled it for `[0,1.8)` because hopefully the not yet released plugin 1.8 will add support. In case there is this section in the pom.xml unconditionally you prevent that potential support from becoming active. In general it is dangerous to disable it for all versions, as you never know which support is added in future versions.

Reply:
I figure this section can just be deleted when the plugin version is bumped to one where support was added. But, if the metadata is added to do anything other than ignore inside the plugin itself, and the user specifies a newer version than what's in the parent POM here, then I guess you are right.